src/NuGet.Core/NuGet.Configuration/PackageSource/NuGetConstants.cs

```diff
@@ -13,7 +13,7 @@ public static class NuGetConstants
         public static readonly string V2LegacyFeedUrl = "https://go.microsoft.com/fwlink/?LinkID=230477";
 
         public static readonly string V1FeedUrl = "https://go.microsoft.com/fwlink/?LinkID=206669";
-        public static readonly string DefaultGalleryServerUrl = "https://www.nuget.org";
+        public static readonly string DefaultGalleryServerUrl = "https://api.nuget.org/v3/index.json";
```

Comment:
Actually, now that I read the notes in the issue, I see why this change was made. I just don't know how big of an effect this URL has.

So, this looks good to me :shipit: